src/core/api.proto

```diff
@@ -132,6 +181,14 @@ message InferRequestHeader
     //@@       highest probabilities will be returned.
     //@@
     Class cls = 3;
+
+    //@@    .. cpp:var:: InferSharedMemory shm_io
+    //@@
+    //@@       It is the location in shared memory to write the output tensors.
```

Comment:
".. location in shared memory that the result tensor data for this output will be written"

shm_io -> shared_memory